index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;700;900&family=Titillium+Web:wght@200;300;400;600;700;900&display=swap"
+        rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <script src="https://kit.fontawesome.com/9f0938f0ba.js" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Nova Theme</title>
+</head>
+
+<body>
+    <!-- *-+*-+*-+*-+*-+*-+*-+*-+*-+ Header Section *-+*-+*-+*-+*-+*-+*-+*-+ -->
```

Comment:
Raczej bym nie używał takich "ślaczków" - w mojej ocenie trochę mnie to rozprasza :D